spec/controllers/checkouts_controller_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+RSpec.describe CheckoutsController, type: :controller do
```

Comment:
Since the rails scaffold generator created a request spec and didn't create a controller spec I was putting the tests in `spec/requests/checkouts_spec.rb`.  The general trend in rspec appears to be towards request specs and away from controller specs (https://medium.com/just-tech/rspec-controller-or-request-specs-d93ef563ef11).  I know we have a ton of controller specs but I thought we could try doing request specs since this is new functionality.  What do you think?

Reply:
I was not aware of that trend. If current best practices/recommendations are to use request specs, we should probably move towards that.